Index: src/tads/MyQueue/Node.java
===================================================================
diff --git a/src/tads/MyQueue/Node.java b/src/tads/MyQueue/Node.java
deleted file mode 100644
--- a/src/tads/MyQueue/Node.java	(revision 9953a6baac42eb44fceaacc9ce851a03bc3ca50f)
+++ /dev/null	(revision 9953a6baac42eb44fceaacc9ce851a03bc3ca50f)
@@ -1,26 +0,0 @@
-public class Node <T>{
-
-    T value;
-    Node next;
-
-    public Node(T value) {
-        this.value = value;
-        this.next = null;
-    }
-
-    public T getValue() {
-        return value;
-    }
-
-    public void setValue(T value) {
-        this.value = value;
-    }
-
-    public Node getNext() {
-        return next;
-    }
-
-    public void setNext(Node next) {
-        this.next = next;
-    }
-}
Index: src/tads/MyLinkedList/Node.java
===================================================================
diff --git a/src/tads/MyLinkedList/Node.java b/src/tads/MyLinkedList/Node.java
deleted file mode 100644
--- a/src/tads/MyLinkedList/Node.java	(revision 9953a6baac42eb44fceaacc9ce851a03bc3ca50f)
+++ /dev/null	(revision 9953a6baac42eb44fceaacc9ce851a03bc3ca50f)
@@ -1,29 +0,0 @@
-package tads.MyLinkedList;
-
-public class Node<B> {
-
-    private B value;
-    private Node<B> next;
-
-    public Node (B value){
-        this.value = value;
-        this.next = null;
-    }
-
-    public B getValue() {
-        return value;
-    }
-
-    public void setValue(B value) {
-        this.value = value;
-    }
-
-    public Node<B> getNext() {
-        return next;
-    }
-
-    public void setNext(Node<B> next) {
-        this.next = next;
-    }
-
-}
Index: src/tads/MyStack/MyStack.java
===================================================================
diff --git a/src/tads/MyStack/MyStack.java b/src/tads/MyStack/MyStack.java
deleted file mode 100644
--- a/src/tads/MyStack/MyStack.java	(revision 9953a6baac42eb44fceaacc9ce851a03bc3ca50f)
+++ /dev/null	(revision 9953a6baac42eb44fceaacc9ce851a03bc3ca50f)
@@ -1,10 +0,0 @@
-package tads.MyStack;
-
-import java.util.EmptyStackException;
-
-public interface MyStack<B> {
-    void push (B element);
-    void pop() throws EmptyStackException;
-    boolean isEmpty();
-    void makeEmpty();
-}
Index: src/tads/MyBST/NodeBST.java
===================================================================
diff --git a/src/tads/MyBST/NodeBST.java b/src/tads/MyBST/NodeBST.java
deleted file mode 100644
--- a/src/tads/MyBST/NodeBST.java	(revision 9953a6baac42eb44fceaacc9ce851a03bc3ca50f)
+++ /dev/null	(revision 9953a6baac42eb44fceaacc9ce851a03bc3ca50f)
@@ -1,47 +0,0 @@
-package tads.MyBST;
-
-public class NodeBST<K extends Comparable<K>,T>{
-    K key;
-    T data;
-
-    NodeBST<K,T> leftChild;
-    NodeBST<K,T> rightChild;
-
-    public NodeBST(K key, T data) {
-        this.key = key;
-        this.data = data;
-    }
-
-    public K getKey() {
-        return key;
-    }
-
-    public void setKey(K key) {
-        this.key = key;
-    }
-
-    public T getData() {
-        return data;
-    }
-
-    public void setData(T data) {
-        this.data = data;
-    }
-
-    public NodeBST<K, T> getLeftChild() {
-        return leftChild;
-    }
-
-    public void setLeftChild(NodeBST<K, T> leftChild) {
-        this.leftChild = leftChild;
-    }
-
-    public NodeBST<K, T> getRightChild() {
-        return rightChild;
-    }
-
-    public void setRightChild(NodeBST<K, T> rightChild) {
-        this.rightChild = rightChild;
-    }
-}
-
Index: src/tads/MyQueue/MyQueueImpl.java
===================================================================
diff --git a/src/tads/MyQueue/MyQueueImpl.java b/src/tads/MyQueue/MyQueueImpl.java
deleted file mode 100644
--- a/src/tads/MyQueue/MyQueueImpl.java	(revision 9953a6baac42eb44fceaacc9ce851a03bc3ca50f)
+++ /dev/null	(revision 9953a6baac42eb44fceaacc9ce851a03bc3ca50f)
@@ -1,71 +0,0 @@
-public class MyQueueImpl<T> implements MyQueue<T>{
-
-    private Node first;
-    private Node last;
-    int size = 0;
-
-    @Override
-    public void enqueue(T element) {
-
-        Node<T> nuevo = new Node(element);
-
-        if(isEmpty())
-        {
-            this.first = nuevo;
-            this.last = nuevo;
-        }else
-        {
-            this.last.setNext(nuevo);
-            this.last = nuevo;
-        }
-        size++;
-    }
-
-    @Override
-    public T dequeue() throws EmptyQueueException {
-
-        Node<T> buscado = null;
-
-        if(isEmpty())
-        {
-            throw new EmptyQueueException();
-        }else
-        {
-            buscado = this.first;
-            this.first = this.first.getNext();
-        }
-
-        return (T) buscado;
-    }
-
-    @Override
-    public boolean isEmpty() {
-        if(this.first==null)
-        {
-            return true;
-        }
-        else
-        {
-            return false;
-        }
-    }
-
-    @Override
-    public boolean contains(T value) {
-
-        boolean contains = false;
-
-        Node<T> temporal = this.first;
-
-        while(temporal!=null && !temporal.getValue().equals(value)){
-            temporal = temporal.getNext();
-        }
-
-        if(temporal!=null)
-        {
-            contains = true;
-        }
-        return contains;
-    }
-}
-
Index: src/tads/MyQueue/MyQueue.java
===================================================================
diff --git a/src/tads/MyQueue/MyQueue.java b/src/tads/MyQueue/MyQueue.java
deleted file mode 100644
--- a/src/tads/MyQueue/MyQueue.java	(revision 9953a6baac42eb44fceaacc9ce851a03bc3ca50f)
+++ /dev/null	(revision 9953a6baac42eb44fceaacc9ce851a03bc3ca50f)
@@ -1,8 +0,0 @@
-public interface MyQueue<T> {
-
-    void enqueue(T element);
-    T dequeue() throws EmptyQueueException;
-    boolean isEmpty();
-    boolean contains(T value);
-
-}
Index: src/tads/MyStack/Node.java
===================================================================
diff --git a/src/tads/MyStack/Node.java b/src/tads/MyStack/Node.java
deleted file mode 100644
--- a/src/tads/MyStack/Node.java	(revision 9953a6baac42eb44fceaacc9ce851a03bc3ca50f)
+++ /dev/null	(revision 9953a6baac42eb44fceaacc9ce851a03bc3ca50f)
@@ -1,28 +0,0 @@
-package tads.MyStack;
-
-public class Node<B> {
-    private B value;
-    private Node<B> next;
-
-    public Node(B value){
-        this.value = value;
-    }
-
-    public Node<B> getNext(){
-        return next;
-    }
-
-    public B getValue(){
-        return value;
-    }
-
-    void setNext(Node<B> next){
-        this.next = next;
-    }
-
-    void setValue(B value){
-        this.value = value;
-    }
-
-
-}
Index: .idea/misc.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<project version=\"4\">\r\n  <component name=\"ProjectRootManager\" version=\"2\" languageLevel=\"JDK_17\" default=\"true\" project-jdk-name=\"17\" project-jdk-type=\"JavaSDK\">\r\n    <output url=\"file://$PROJECT_DIR$/out\" />\r\n  </component>\r\n</project>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/misc.xml b/.idea/misc.xml
--- a/.idea/misc.xml	(revision 9953a6baac42eb44fceaacc9ce851a03bc3ca50f)
+++ b/.idea/misc.xml	(date 1654985222688)
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_17" default="true" project-jdk-name="17" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_17" project-jdk-name="17" project-jdk-type="JavaSDK">
     <output url="file://$PROJECT_DIR$/out" />
   </component>
 </project>
\ No newline at end of file
Index: .idea/modules.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<project version=\"4\">\r\n  <component name=\"ProjectModuleManager\">\r\n    <modules>\r\n      <module fileurl=\"file://$PROJECT_DIR$/harTADs.iml\" filepath=\"$PROJECT_DIR$/harTADs.iml\" />\r\n    </modules>\r\n  </component>\r\n</project>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/modules.xml b/.idea/modules.xml
--- a/.idea/modules.xml	(revision 9953a6baac42eb44fceaacc9ce851a03bc3ca50f)
+++ b/.idea/modules.xml	(date 1654985387928)
@@ -2,7 +2,8 @@
 <project version="4">
   <component name="ProjectModuleManager">
     <modules>
-      <module fileurl="file://$PROJECT_DIR$/harTADs.iml" filepath="$PROJECT_DIR$/harTADs.iml" />
+      <module fileurl="file://$PROJECT_DIR$/grupo13-p2-tads/grupo13-p2-tads.iml" filepath="$PROJECT_DIR$/grupo13-p2-tads/grupo13-p2-tads.iml" />
+      <module fileurl="file://$PROJECT_DIR$/.idea/harTADs.iml" filepath="$PROJECT_DIR$/.idea/harTADs.iml" />
     </modules>
   </component>
 </project>
\ No newline at end of file
